ota-plus-web/app/reactapp/src/components/campaigns/Wizard.jsx

```diff
@@ -133,6 +133,47 @@ class Wizard extends Component {
   @observable rawSelectedPacks = [];
   @observable approvalNeeded = false;
   versions = {};
+
```

Comment:
Let's avoid moving code in bugfix PRs. Should move code in separate commit and separate PR. Bugfix PRs should be as small as possible.

Reply:
Reverted